containers-tests/benchmarks/Graph.hs

```diff
@@ -74,3 +75,9 @@ buildLineG :: Int -> Graph
 buildLineG n = makeG label (1, n) (zip [1..n-1] [2..])
   where
     label = "line,n=" ++ show n
+
+-- A maximal DAG. There is an edge from vertex i to j for every i, j when i < j.
+buildMaxDAG :: Int -> Graph
+buildMaxDAG n = makeG label (1, n) [(i, j) | i <- [1..n], j <- [i+1 .. n]]
```

Comment:
When `i=n`, there's no room for a `j`. I think it would be clearer to write `i <- [1..n-1]`.

Reply:
Done.